CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 ### Changed
 - Move npm tools and config to default locations (#444)
 - Use the TYPO3 Code of Conduct (#430)
+- Change the default indentation for rst files to 4 spaces (https://github.com/TYPO3-Documentation/T3DocTeam/issues/194)
```

Comment:
Now that 2.0.0 is out, this needs to go into the changelog for the upcoming version (x.y.z), not 2.0.0 anymore.

Reply:
done